third_party/CMakeLists.txt

```diff
@@ -1,73 +1,6 @@
-include(ExternalProject)
+option(TTMLIR_ENABLE_ONNX_MLIR "Enable ONNX MLIR support" OFF)
```

Comment:
We should move this option to the top level `CMakeLists.txt` and also enable it in the "Configure CMake" step of `.github/workflows/build.yml` so that CI can test onnx conversion.

Reply:
Sure I can move it to CMakeLists.txt. For adding to build.yml, I suggest we delay it until we can resolve LLVM version dependency, as the LLVM version might need to changed so a version that works everything e.g. both ONNX-MLIR and StableHLO.